include/k4ainternal/common.h

```diff
@@ -24,7 +24,8 @@ typedef struct _guid_t
 #define MAX_FPS_IN_MS (33) // 30 FPS
 
 #define COUNTOF(x) (sizeof(x) / sizeof(x[0]))
-
+#define MIN(a, b) ((a) < (b) ? (a) : (b))
+#define MAX(a, b) ((a) > (b) ? (a) : (b))
```

Comment:
To avoid double evaluation bugs in the future:

```suggestion
#define MAX(a, b) \
({ __typeof__ (a) _a = (a); \
   __typeof__ (b) _b = (b); \
   (_a > _b) ? _a : _b; })
```